code/modules/surgery/organs/tongue.dm

```diff
@@ -40,6 +43,11 @@
 
 /obj/item/organ/tongue/Insert(mob/living/carbon/tongue_owner, special = 0)
 	..()
+	if(tongue_owner.dna && tongue_owner.dna.species)
+		tongue_owner.dna.species.say_mod = say_mod
+		tongue_owner.dna.species.ask_mod = ask_mod
+		tongue_owner.dna.species.exclaim_mod = exclaim_mod
+		tongue_owner.dna.species.yell_mod = yell_mod
```

Comment:
Wouldn’t it be easier to delegate the species say mods to whatever tongue they are created with, and then have the speech code directly use the tongues speech mod? That way you don’t have to keep overriding the species datum (which you is not meant to be overridden), the speech wont revert on species change and the code would be a lot more future proof

Reply:
i would do that if i had enough brain cells to code it but unfortunately i'm a fucking idiot